bionic/task_state.py

```diff
@@ -339,21 +336,39 @@ def initialize(self, bootstrap, flow_instance_uuid):
             for task_key in self.task_keys
         ]
 
+        should_persist = (
+            single_unique_element(
+                entity_def.can_persist
+                for entity_def in self.entity_defs_by_dnode.values()
+            )
+            and not self.output_would_be_missing()
+        )
+        if should_persist and bootstrap is None:
+            descriptors = [
+                task_key.dnode.to_descriptor() for task_key in self.task_keys
+            ]
+            if self.task.is_simple_lookup:
+                disable_message = """
+                applying `@persist(False)` or `@immediate` to your corresponding
+                function"""
+            else:
+                disable_message = """
+                passing `persist=False` argument when you `declare` / `assign` your
```

Comment:
nit: "passing `persist=False` argument" is not quite grammatically right, since it's missing a determiner. I think we can just omit "argument".

Also, I would use "the entity value" instead of "your values".